rules-reviewed/README.txt

```diff
@@ -0,0 +1,14 @@
+This directory contains default set of Windup rules distributed with RHAMT.
```

Comment:
Good idea... although maybe once this current round of fixes is done, we can fix the name. rules-reviewed shouldn't stay here forever, IMO.